src/templates/layouts/pressRelease/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { PressRelease as FormattedPressRelease } from '@/types/formatted/pressRelease'
+import { LovellStaticPropsResource } from '@/lib/drupal/lovell/types'
+import { LovellSwitcher } from '@/templates/components/lovellSwitcher'
+import { formatDate } from '@/lib/utils/helpers'
+import { recordEvent } from '@/lib/analytics/recordEvent'
+import { ContentFooter } from '@/templates/common/contentFooter'
+
+export const PressRelease = ({
+  title,
+  releaseDate,
+  pdfVersion,
+  introText,
+  address,
+  fullText,
+  contacts,
+  downloads,
+
+  /*listing,*/
+  lovellVariant,
+  lovellSwitchPath,
+}: LovellStaticPropsResource<FormattedPressRelease>) => {
+  return (
+    <>
+      <div>
+        <main className="va-l-detail-page va-facility-page">
+          <div className="usa-grid usa-grid-full">
+            {/* nav here */}
+            <div className="usa-width-three-fourths">
+              <article className="usa-content">
+                <LovellSwitcher
+                  currentVariant={lovellVariant}
+                  switchPath={lovellSwitchPath}
+                />
+                <section className="vads-u-margin-bottom--5">
+                  <h1 className="vads-u-margin-bottom--2p5">{title}</h1>
+                  <p className="vads-u-margin-bottom--0p5">PRESS RELEASE</p>
+                  <p className="vads-u-font-weight--bold vads-u-margin-bottom--3 vads-u-margin-top--0">
+                    {releaseDate || formatDate(releaseDate)}
+                  </p>
+                  <div>
+                    <button
+                      type="button"
+                      className="va-u-margin-right--4 va-button-link"
+                      onClick={() => {
+                        window.print()
+                        return false
+                      }}
+                    >
+                      <i className="fa fa-print vads--u-padding-right--1"></i>
+                      Print
+                    </button>
+                    {pdfVersion && (
+                      <a href={pdfVersion.field_document.uri.url} download>
+                        <i className="fa fa-download vads--u-padding-right--1"></i>
+                        Download press release (PDF)
+                      </a>
+                    )}
+                  </div>
+                  <p className="va-introtext vads-u-font-size--lg vads-u-margin-top--3">
+                    {address.locality},{address.administrative_area} -
+                    {introText}
+                  </p>
+                  {/* Body */}
+                  <div dangerouslySetInnerHTML={{__html: fullText.processed}}></div>
+                </section>
+                <section className="vads-u-margin-bottom--6">
+                  <div className="vads-u-font-weight--bold">Media contacts</div>
+                  {/* Print each media contact */}
+                  {contacts.map((contact) => {
```

Comment:
I would break this out to an additional component (and normal away the drupal field names in the formatter). 

The component could be an additional function inside of `contactInfo.tsx`, perhaps `PressReleaseContact`.

The formatter should turn `field_press_release_contact` into some kind of `PressContact` type.

```js
{
   id. entity.id,
   name: entity.title,
   job: entity.field_description,
   phone: entity.field_phone_number,
   email: entity.field_email_address
}
```